torch_geometric/graphgym/models/layer.py

```diff
@@ -156,18 +223,21 @@ class MLP(nn.Module):
         num_layers (int): Number of layers in the stack
         **kwargs (optional): Additional args
     """
-    def __init__(self, dim_in, dim_out, bias=True, dim_inner=None,
-                 num_layers=2, **kwargs):
+    def __init__(self, layer_config: LayerConfig, **kwargs):
```

Comment:
E.g., has_batchnorm does not apply here, but a user may not know that, and still set `has_batchnorm` and expect it to work.